index.html

```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>    
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
+    <div class="works">
+        <div class="recent">
+            <div class="title">
+                <h2 class="first">My Recent Works</h2>
+            </div>            
+            <hr class="first-line">
+        </div>
+            
+        <div class="cards">
+           <div class="one">
+                    <div class="this-pic">
+                        <img class="picture" src="image/ImgPlaceholder.png" alt="icon">
+                   </div>
+             <div class="post">
+                <h3 class="multi">Multi-Post Stories</h3>
+                <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <div class="languages">
+                    <ul>
+                        <li class="css">css<a href="#"></a></li>
+                        <li class="html">html<a href="#"></a></li>
+                        <li class="bootstrap">bootstrap<a href="#"></a></li>
+                        <li class="ruby">ruby<a href="#"></a></li>
+                        </ul>
+                </div>
+                <div><button  type="button" class="first-btn">See Project</button></div>
+              </div>
```

Comment:
- [ ] The description of each of these projects should be aligned to the bottom of the image. kindly fix that

Figma design          |  your design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/188262758-d9b15435-567f-4c64-83b8-5a3db086cadf.png)  |  ![image](https://user-images.githubusercontent.com/89282221/188262739-d25a384b-4b22-4110-a29f-1cf8c8e82faf.png)